Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2290,6 +2290,10 @@ extension BrowserViewController: TabManagerDelegate {
             topTabsDidChangeTab()
         }
 
+        if let url = selected?.url, let internalUrl = InternalURL(url), internalUrl.originalURLFromErrorPage != nil {
```

Comment:
that looks better thanks for making the change, I found that `InternalUR` has a computed var `isErrorPage` I think using it here will make the intention clearer. We want to know if the tab is an errorPage to reload it. Could you add a comment too please 🙏 something like:
```suggestion
       /// If the selectedTab is showing an error page trigger a reload
        if let url = selected?.url, let internalUrl = InternalURL(url), internalUrl.isErrorPage {
```
I will trigger the build to get the process started